src/Service/Font/FontLoader.re

```diff
@@ -57,7 +58,7 @@ let loadAndValidateEditorFont =
               requestId,
               {
                 fontFile: fullPath,
-                fontFamily: Revery.Font.Family.fromFile(fullPath),
+                fontFamily: Revery.Font.Family.system(familyName),
```

Comment:
It looks like this logic is unfortunately broken in a couple cases:
- When no `editor.fontFamily` is specified (ie, the default case) - we should make sure we're loading our default font.
- When a path is specified, ie `/path/to/some-font.ttf` - we should handle that case, and just have a custom resolver that always resolves to the font file.

Reply:
I think I fixed this in the latest commit. Thanks for pointing it out! Let me know if anything is still broken that you can find. I wasn't able to.